flow/platforms/sky130hd/chameleon/chameleon.macro_placment.cfg

```diff
@@ -1,6 +1,6 @@
-_4672_ R0 200 150
```

Comment:
If you added flow/designs/sky130hd/chameleon/chameleon.macro_placment.cfg why do we still need this?

Reply:
I rm'd the platform copy.